### DISCUSSION TITLE
Location view: Devices of nested locations

### DISCUSSION THREAD

**Discussion by dreng** [netbox-community/netbox#6225] [Q&A]
Hi all,

I'm a bit confused. Maybe it is just caused by misunderstanding but since I got no reply I'm not sure what to do.

As stated in #6124 (see my last comment) that feature has not been implemented although it has been accepted and some code has been committed. Devices of nested locations are still not present in a location view. See this:

![image](https://user-images.githubusercontent.com/20901110/115445896-be436480-a216-11eb-95a7-e345c7ba8a1a.png)

But that:
![image](https://user-images.githubusercontent.com/20901110/115445957-cf8c7100-a216-11eb-8011-9c7058217105.png)

Has this been noticed? Has my comment been ignored on purpose? Should I file a new bug or feature request?

Thanks in advance for clarification.

**Comment by ryanmerolle** [comment]
I think this was address in v2.11 correct?
